microcosm_caching/memcached.py

```diff
@@ -33,9 +34,9 @@ def json_serializer(key, value):
 
     """
     if isinstance(value, str) or isinstance(value, bytes):
-        return value, SerializationFlag.STRING
+        return value, SerializationFlag.STRING.value
 
```

Comment:
Out of curiosity why change this?

Reply:
What ends up happening is that pymemcache ends up sending exactly what it's told to send in (won't try to convert any values). As such, the final combined command was looking something like this:
```
set sphinx:a935d5b1-205e-4b2b-b204-0b42fe98f55c SerializationFlag.JSON 0 969\r\n{"factType":...
```
Note that the `Flag` param in there should be `2`, not the enum itself.

Reply:
https://pymemcache.readthedocs.io/en/latest/getting_started.html#serialization